pallets/parachain-staking/src/lib.rs

```diff
@@ -179,6 +179,9 @@ pub mod pallet {
 	/// The current storage version.
 	const STORAGE_VERSION: StorageVersion = StorageVersion::new(8);
 
+	/// The 5.1% inflation rate of the third year
+	const INFLATION_3RD_YEAR: u64 = 51_000_000_000_000_000;
```

Comment:
It would be clearer if the entire value is defined together. Otherwise this very long number is not very explicative until it's seen used in a `Perquintill` context, down the line.